tests/services/test_ntfy.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# (c) 2021-2023 The mqttwarn developers
+from unittest import mock
+from unittest.mock import call
+
+from mqttwarn.model import ProcessorItem as Item
+from mqttwarn.util import load_module_by_name
+
+
+@mock.patch("apprise.Apprise", create=True)
+@mock.patch("apprise.AppriseAsset", create=True)
+def test_ntfy_success(apprise_asset, apprise_mock, srv, caplog):
+    module = load_module_by_name("mqttwarn.services.apprise_multi")
+
+    item = Item(
+        addrs=[
+            {
+                "baseuri": "ntfy://user:password@ntfy.example.org/topic1/topic2?email=test@example.org",
+            }
+        ],
+        title="⚽ Message title ⚽",
+        message="⚽ Notification message ⚽",
+        data={"priority": "high", "tags": "foo,bar", "click": "https://httpbin.org/headers"},
```

Comment:
However, @zoic21 outlined at https://github.com/jpmens/mqttwarn/issues/607#issuecomment-1427813040, that they would like to submit `tags` as a `list`? Please raise your voice if this would be important to you, then we can try to add a corresponding translator.

Reply:
Hello,
No string to string is better, I used array because it's mandatory by ntfy when we used json but it's no necessary by apprise